en/docs/administer/advanced-configurations.md

```
@@ -13,7 +13,7 @@ This section covers how to change and configure several features related to the
 
 1. Change the relevant configuration accordingly and click **Save**.
 
-    <a href="{{base_path}}/assets/img/administer/advanced-config-save.png"><img src="{{base_path}}/assets/img/administer/advanced-config-save.png"/></a>
+    <a href="https://<your-docs-domain>/assets/img/learn/.../assets/img/administer/advanced-config-save.png"><img src="https://<your-docs-domain>/assets/img/learn/.../assets/img/administer/advanced-config-save.png"/></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)